### PR TITLE
Support for Travis CI

### PR DESCRIPTION
**Pull request by romanpeters**
### Travis CI

This PR adds support for Travis CI, using a `.travis.yml` file.

Things still needed before merging:
 - [x] Request accepted by repo owner for https://travis-ci.com
 - [x] Create a new Discord API key as an encrypted file or an environment variable and add it to `.travis.yml`.
 - [ ] Add a fancy Travis badge to the README

### CODE REVIEW

**Comment by RobinSikkens** [comment]
Travis is obsolete as code review is already done by Codacy.

**Comment by romanpeters** [comment]
Integration testing ≠ code quality testing.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+- 3.6
+before_install:
+- touch .env
```
Comment (romanpeters) [inline .travis.yml:5]: I suggest using a new API key just for Travis and adding it using `travis encrypt TOKEN="123456789" --add`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+install:
+- python setup.py install
+script:
+- start
```
Comment (romanpeters) [inline .travis.yml:9]: These are the steps taken from the README, but apparently it doesn't work yet. Are there more steps to take in order to run the bot?

Comment (RobinSikkens) [reply .travis.yml:9]: Thats shouldn't be the case. According to the Stacktrace it might be a problem with line-endings

**Pull request merged**